script/passenger-recycler.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+#
+# Trivial Passenger memory monitor installed via /etc/cron.hourly/. To modify
+# configuration create /etc/foreman/passenger-recycler.rb.conf file.
+#
+
+CONFIG = '/etc/foreman/passenger-recycler.rb.conf'
+load CONFIG if File.readable?(CONFIG)
+ENABLED ||= true
+MAX_PRIV_RSS_MEMORY ||= 2_000_000
+GRACEFUL_SHUTDOWN_SLEEP ||= 120
+KILL_BUSY ||= true
+SEND_STATUS ||= true
+exit 0 unless ENABLED
+
+def running?(pid)
+  return Process.getpgid(pid) != -1
+rescue Errno::ESRCH
+  return false
+end
+
+require 'phusion_passenger'
```

Comment:
This assumes the Passenger installation's been added to the load path, how will it load from non-rubygems packages, e.g. /usr/share/passenger? Should it also set the path to locations.ini?